app/models/request.rb

```diff
@@ -21,8 +21,9 @@ class Request < ApplicationRecord
   enum status: %i[pending accepted rejected]
   validates :name,
             length: { maximum: MAX_NAME_LENGTH, message: 'only allows a maximum of %{count} characters' },
-            format: { with: /\A[a-z0-9\-]+\z/, message: 'only letters and numbers allowed' },
+            format: { with: /\A[a-z0-9\-]+\z/, message: 'only allows lowercase letters, numbers and "-"' },
             uniqueness: true
```

Comment:
Don't validate uniqueness here, rejected request names should become available again